### PR TITLE
raylib: fix problems with glfw version

### PR DESCRIPTION
**Pull request by Julianiolo**
Specify library name and version:  **raylib/5.0**

This fixes a problem with the glfw version. Raylib needs glfw/3.4, but currently we use glfw/3.3.8.
Raylib requires version 3.4 in cmake ([raylib](https://github.com/raysan5/raylib/blob/d9c5066382615644137b4f65479c65c44820027a/cmake/GlfwImport.cmake#L3)) but it somehow still manages to compile? Raylib also uses features of 3.4, such as new mouse cursors. But when building this into a application glfw gives errors, because it does not know these new mouse cursors.
Setting the required glfw to glfw/3.4 (as this PR does) seems to fix all problems. But maybe it should be investigated, why this could build?

TLDR: raylib seems to get compiled with glfw/3.4, although glfw/3.3.8? Raylib requires 3.3.8.

---

- [ x ] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ x ] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by Julianiolo** [comment]
for the cursors, see:

[3.8.8](https://github.com/glfw/glfw/blob/7482de6071d21db77a7236155da44c172a7f6c9e/include/GLFW/glfw3.h#L1078)

[3.4](https://github.com/glfw/glfw/blob/7b6aead9fb88b3623e3b3725ebb42670cbe4c579/include/GLFW/glfw3.h#L1197)

Produced errors like this:
```
WARNING: GLFW: Error: 65539 Description: Invalid standard cursor 0x00036009
WARNING: GLFW: Error: 65539 Description: Invalid standard cursor 0x00036008
WARNING: GLFW: Error: 65539 Description: Invalid standard cursor 0x00036007
WARNING: GLFW: Error: 65539 Description: Invalid standard cursor 0x0003600A
```

There also were some error messages on raylib startup:
```
...
INFO:     > rmodels:... loaded (optional)
INFO:     > raudio:.... loaded (optional)
WARNING: GLFW: Error: 65539 Description: Invalid window hint 0x0002000D
INFO: DISPLAY: Trying to enable VSYNC
INFO: DISPLAY: Device initialized successfully
INFO:     > Display size: 1920 x 1080
INFO:     > Screen size:  800 x 600
INFO:     > Render size:  800 x 600
INFO:     > Viewport offsets: 0, 0
INFO: GLAD: OpenGL extensions loaded successfully
...
```

**Review by RubenRBS — APPROVED** [review]
Thanks a lot for such a detailed description, this looks great, we appreciate it :)
